index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>CR Trials</title>
+</head>
+<body>
+    <h1 class="header">Hello Microverse</h1>
```

Comment:
- [ ] It would be good to use the `<header>` tag for creating your header. This will make your project even more semantic and hence making your site SEO friendly. You can learn more about semantic HTML [here](https://www.w3schools.com/html/html5_semantic_elements.asp)